onboarding_project/squad_pantry_app/models.py

```diff
@@ -19,31 +24,59 @@ class Dish(models.Model):
         validators=[MinValueValidator(1)], help_text='Time Taken to Prepare the Dish')
 
     def __str__(self):
-        return self.dish_name
+        return self.dish_name+" ("+str(self.prep_time_in_minutes)+" Min)"
```

Comment:
We favor string formatting over string concatenation. So this should have been
`"{0} ({1} Min)".format(self.dish_name, self.prep_time_in_minutes)`.
Makes the code more readable.

Reply:
Done!